staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/admission.go

```diff
@@ -195,7 +153,7 @@ func (a *GenericAdmissionWebhook) ValidateInitialization() error {
 	if a.hookSource == nil {
 		return fmt.Errorf("the GenericAdmissionWebhook admission plugin requires a Kubernetes client to be provided")
 	}
-	if a.negotiatedSerializer == nil {
+	if a.clientManager.GetNegotiatedSerializer() == nil {
```

Comment:
Maybe define a `func (c *ClientManager) Valid() error` and call it here. Shouldn't this be checking the resolvers, too?

Reply:
Fixed.